runtime/src/main/java/org/corfudb/runtime/view/stream/BackpointerStreamView.java

```diff
@@ -2,7 +2,10 @@
 
 import com.google.common.annotations.VisibleForTesting;
 
+import java.util.ArrayList;
 import java.util.Arrays;
+import java.util.Collections;
```

Comment:
I think we should keep this implementation as is. Maybe you can introduce a new implementation.

Reply:
Done.